### DISCUSSION TITLE
Not inputting keypresses when keys are pressed.

### DISCUSSION THREAD

**Discussion by GamingWithColby** [ppy/osu#17442] [Q&A]
### Type

Game behaviour

### Bug description

Yes that title was the only good description I could give it. But when doing any songs it sometimes wont use the button presses I input, Although the in game counter definitely says I'm pressing them, I've messed with in game settings for a good few hours now, and have made it slightly better, I had to verify it wasn't me by using the autopilot mod. 

### Screenshots or videos

It is very difficult to get a video, showing it's not a skill issue. all I can say is I do not have this problem on cutting edge osu.

### Version

2022.319.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/8347204/database.log)
[network.log](https://github.com/ppy/osu/files/8347205/network.log)
[performance.log](https://github.com/ppy/osu/files/8347206/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8347207/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8347208/updater.log)

**Comment by peppy** [comment]
You're going to have to provide more information than this for us to help you in any way. A video would be the minimal requirement, I think.
